site/team/laufen/index.html

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+        <div class="container">
+        </br>
+        </br>
+        </br>
+        </br>
```

Comment:
?

This does not change anything, remove the /br tags.


To add some space you can change the container to "container distance"